src/community/jwt-headers/jwt-headers-util/src/main/java/org/geoserver/security/jwtheaders/token/TokenSignatureValidator.java

```diff
@@ -50,7 +50,9 @@ public JWKSet load(String urlStr) throws Exception {
 
     public static JWKSet loadJWKSet(String urlStr) throws Exception {
         URL url = new URL(urlStr);
-        return JWKSet.load(url);
+        try (var stream = url.openStream()) {
```

Comment:
Huh, it does not use the http client event.

@davidblasby (or @david-blasby) this is your community module so you do not need to wait on PRs while under development.